Yoast/YoastSEO.js
# Examples webpack work

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

- Fixed some spacing irregularities with the paper UI, columns and inputs.
- Fixed Jed error from `yoast-components`.
- Fixed `locale` not being saved to the configuration.
- Added webpack alias to the `YoastSEO.js` `src` folder.
- Added `ButtonContainer`, `HeadingContainer` and `HorizontalContainer`.
- Added `fullTestTexts`'s papers as input with the help of `react-select`.
- Added overall scores.
- Changed raw markings: moved inside a collapsible to save screen space.
- Changed control buttons to `yoast-components`'s `IconButton`.
- Changed `StatusBox` to use flexbox.
- Removed TypeScript loader.
- Removed the H1, this is not an actual web page; it took up valuable screen space.

## Test instructions

This PR can be tested by following these steps:

* Go to `examples/webpack`.
* Install dependencies: `yarn`.
* Run it: `yarn start`.
* Check out the changes (see the list in the choices section).
* Ensure other things are still functioning properly.
